CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+## How to Contribute
+
+### Raising an issue:
+ This is an Open Source project and we would be happy to see contributors who report bugs and file feature requests submitting pull requests as well.
+ This project adheres to the Contributor Covenant code of conduct.
+ By participating, you are expected to uphold this code style.
+ Please report issues here [Issues - fossasia/open-event-android](https://github.com/fossasia/open-event-android/issues)
+
+### Branch Policy
+
+#### Sending pull requests:
+
+Say you want to contribute changes to someone else’s repository (eg, this one).
+
+It is advisable to clone only the development branch using following command:
+Go to the repository on github. (This repository is by fossasia, and is called open-event-android, then you’ll find it at http://github.com/fossasia/open-event-android.)
+
+
+Click the “Fork” button at the top right.
+
+You’ll now have your own copy of that repository in your github account.
+
+Open a terminal/shell.
+
+Type
+
+`$ git clone git@github.com:username/open-event-android`
+
+where 'username' is your username.
+
+You’ll now have a local copy of your version of that repository.
+
+*For further details on how to contribute as a first time contributer visit [Contribute to someone's repository](http://kbroman.org/github_tutorial/pages/fork.html)*
+
+#### Change into that project directory (open-event-android):
+
+`$ cd open-event-android`
+
+#### Add a connection to the original owner’s repository.
+
+`$ git remote add foss_remote git://github.com/fossasia/open-event-android`
+
+Note the distinction between git@github.com: in the first case and git://github.com/ in the second case. I’m not sure why these need to be the way they are, but that’s what works for me.
```

Comment:
Remove, because both will work. The first syntax is just doing via SSH. Also, I'd recommend to change both commands to HTTPS versions as that's what will work for beginners who might not have added their SSH keys to Github